x-pack/legacy/plugins/apm/public/components/shared/DottedKeyValueTable/index.tsx

```diff
@@ -59,14 +59,18 @@ export function DottedKeyValueTable({
   data,
   parentKey,
   maxDepth,
-  tableProps = {}
+  tableProps = {},
+  skipPathify = false
 }: {
-  data: StringMap;
+  data: StringMap | PathifyResult;
```

Comment:
We should get rid of StringMap since it's basically `Record<string, T>`. Therefore this can be written as:

```suggestion
  data: Record<string, any>;
```

Reply:
Btw. I'm not sure we want to use `any` here. Shouldn't it instead be:
```
Record<string, unknown>
```
?